src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -254,4 +257,29 @@ public String getJavaHomeDir() {
   public void setJavaHomeDir(String javaHomeDir) {
     this.javaHomeDir = javaHomeDir;
   }
+
+  @Override
+  public ImmutableMap<String, String> getEnvironmentVariables() {
+    return environment;
+  }
+ 
+  /**
+   * Makes an immutable copy of the specified map. 
+   * 
+   * @param environment the environment variables to store.
+   * @throws NullPointerException if any key or value in environment is null
+   * @throws IllegalArgumentException if any key contains the NUL or = character 
+   */
+  public void setEnvironmentVariables(Map<String, String> environment) {
```

Comment:
`addEnvironmentVariable(s)` suddenly came up in my mind, but `setEnvironmentVariables` LGTM.